### ISSUE TITLE
Image failing on start

### ISSUE THREAD

**Issue by sepharg** [tiangolo/uwsgi-nginx-flask-docker#140]
Since the new change on nginx.conf we added the following line to our Dockerfile

`COPY ./nginx.conf /app/nginx.conf`

but now on startup we get the following errors
any ideas??

6/14/2019 4:06:30 PMworker 1 buried after 1 seconds
6/14/2019 4:06:30 PMworker 2 buried after 1 seconds
6/14/2019 4:06:30 PMgoodbye to uWSGI.
6/14/2019 4:06:32 PM/usr/lib/python2.7/dist-packages/supervisor/options.py:298: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
6/14/2019 4:06:32 PM  'Supervisord is running as root and it is searching '
6/14/2019 4:06:33 PM[uWSGI] getting INI configuration from /app/uwsgi.ini
6/14/2019 4:06:33 PM[uWSGI] getting INI configuration from /etc/uwsgi/uwsgi.ini
6/14/2019 4:06:33 PM
6/14/2019 4:06:33 PM;uWSGI instance configuration
6/14/2019 4:06:33 PM[uwsgi]
6/14/2019 4:06:33 PMcheaper = 2
6/14/2019 4:06:33 PMprocesses = 16
6/14/2019 4:06:33 PMini = /app/uwsgi.ini
6/14/2019 4:06:33 PMmodule = app.main
6/14/2019 4:06:33 PMcallable = app
6/14/2019 4:06:33 PMenable-threads = true
6/14/2019 4:06:33 PMini = /etc/uwsgi/uwsgi.ini
6/14/2019 4:06:33 PMsocket = /tmp/uwsgi.sock
6/14/2019 4:06:33 PMchown-socket = nginx:nginx
6/14/2019 4:06:33 PMchmod-socket = 664
6/14/2019 4:06:33 PMhook-master-start = unix_signal:15 gracefully_kill_them_all
6/14/2019 4:06:33 PMneed-app = true
6/14/2019 4:06:33 PMdie-on-term = true
6/14/2019 4:06:33 PMshow-config = true
6/14/2019 4:06:33 PM;end of configuration
6/14/2019 4:06:33 PM
6/14/2019 4:06:33 PM*** Starting uWSGI 2.0.18 (64bit) on [Fri Jun 14 14:06:33 2019] ***
6/14/2019 4:06:33 PMcompiled with version: 6.3.0 20170516 on 16 May 2019 03:07:24
6/14/2019 4:06:33 PMos: Linux-4.4.0-133-generic #159-Ubuntu SMP Fri Aug 10 07:31:43 UTC 2018
6/14/2019 4:06:33 PMnodename: 32988a72b7a3
6/14/2019 4:06:33 PMmachine: x86_64
6/14/2019 4:06:33 PMclock source: unix
6/14/2019 4:06:33 PMpcre jit disabled
6/14/2019 4:06:33 PMdetected number of CPU cores: 4
6/14/2019 4:06:33 PMcurrent working directory: /app
6/14/2019 4:06:33 PMdetected binary path: /usr/local/bin/uwsgi
6/14/2019 4:06:33 PMyour memory page size is 4096 bytes
6/14/2019 4:06:33 PMdetected max file descriptor number: 1048576
6/14/2019 4:06:33 PMlock engine: pthread robust mutexes
6/14/2019 4:06:33 PMthunder lock: disabled (you can enable it with --thunder-lock)
6/14/2019 4:06:33 PMuwsgi socket 0 bound to UNIX address /tmp/uwsgi.sock fd 3
6/14/2019 4:06:33 PMuWSGI running as root, you can use --uid/--gid/--chroot options
6/14/2019 4:06:33 PM*** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
6/14/2019 4:06:33 PMPython version: 3.6.8 (default, May  8 2019, 05:35:00)  [GCC 6.3.0 20170516]
6/14/2019 4:06:33 PMPython main interpreter initialized at 0x5585518b4390
6/14/2019 4:06:33 PMuWSGI running as root, you can use --uid/--gid/--chroot options
6/14/2019 4:06:33 PM*** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
6/14/2019 4:06:33 PMpython threads support enabled
6/14/2019 4:06:33 PMyour server socket listen backlog is limited to 100 connections
6/14/2019 4:06:33 PMyour mercy for graceful operations on workers is 60 seconds
6/14/2019 4:06:33 PMmapped 1239640 bytes (1210 KB) for 16 cores
6/14/2019 4:06:33 PM*** Operational MODE: preforking ***
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'acceptability.models.elmo_classifier.ELMOClassifier' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.dropout.Dropout' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.sparse.Embedding' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.rnn.LSTM' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.linear.Linear' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.container.ModuleList' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.activation.ReLU' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.activation.Sigmoid' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PM/usr/local/lib/python3.6/site-packages/torch/serialization.py:425: SourceChangeWarning: source code of class 'torch.nn.modules.activation.Softmax' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
6/14/2019 4:06:51 PM  warnings.warn(msg, SourceChangeWarning)
6/14/2019 4:06:51 PMWSGI app 0 (mountpoint='') ready in 18 seconds on interpreter 0x5585518b4390 pid: 16 (default app)
6/14/2019 4:06:51 PMuWSGI running as root, you can use --uid/--gid/--chroot options
6/14/2019 4:06:51 PM*** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
6/14/2019 4:06:51 PM*** uWSGI is running in multiple interpreter mode ***
6/14/2019 4:06:51 PMspawned uWSGI master process (pid: 16)
6/14/2019 4:06:51 PMspawned uWSGI worker 1 (pid: 24, cores: 1)
6/14/2019 4:06:51 PMspawned uWSGI worker 2 (pid: 25, cores: 1)
6/14/2019 4:06:51 PMrunning "unix_signal:15 gracefully_kill_them_all" (master-start)...

**Comment by tiangolo** [comment]
I imagine you solved your problem, so thanks for coming back and closing the issue :+1:
